board/batocera/fsoverlay/etc/init.d/S11share

```diff
@@ -127,12 +127,12 @@ EOF
 # wifi configuration is related to the share mounting
 # because some share mounts may depend on the wifi
 # because wifi password are on the share
-rb_wifi_configure_all() {
+batocera_wifi_configure_all() {
     rfkill unblock wifi
 
-    rb_wifi_configure 1&  # 0.6
-    rb_wifi_configure 2&  # 0.6
-    rb_wifi_configure 3&  # 0.6
+    batocera_wifi_configure 1&  # 0.6
+    batocera_wifi_configure 2&  # 0.6
+    batocera_wifi_configure 3&  # 0.6
     wait
```

Comment:
wait is useless without argument, can be removed. Seems an old residue

Reply:
Fixed